viz-lib/src/visualizations/counter/Renderer.tsx

```diff
@@ -10,17 +10,19 @@ import "./render.less";
 
 function getCounterStyles(scale: any) {
   return {
-    msTransform: `scale(${scale})`,
-    MozTransform: `scale(${scale})`,
-    WebkitTransform: `scale(${scale})`,
-    transform: `scale(${scale})`,
+    fontSize: scale,
   };
 }
 
 function getCounterScale(container: any) {
-  const inner = container.firstChild;
-  const scale = Math.min(container.offsetWidth / inner.offsetWidth, container.offsetHeight / inner.offsetHeight);
-  return Number(isFinite(scale) ? scale : 1).toFixed(2); // keep only two decimal places
+  // size of font in base container
+  // children use a relative font size (em)
+  if (container.clientHeight < 50)
```

Comment:
Maybe fixed value is not a better way.

Reply:
Oh, that's an interesting idea.  That would probably be more flexible.

We can try it I guess? :smile:

Reply:
@gaecoli I adjusted the logic to scale the text using the container height, but with fixed upper and lower boundaries. Let me know what you think of this.